src/main/kubernetes/artemis/ingresses/activemq-artemis-ingress.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
```

Comment:
Why are you using the beta version for the ingress?

Reply:
similar to others - default config, changed it to networking.k8s.io/v1